### PR TITLE
reset password by admin

### PR DESCRIPTION
**Pull request by RavindraDesai042**

### CODE REVIEW

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/WEB-INF/controller.xml [hunk 1]
```diff
@@ -254,6 +254,12 @@
 		<response name="success" type="view" value="resetPasswordResponse"/>
 		<response name="error" type="view" value="initResetPassword" />
 	</request-map>
+	<request-map uri="initResetPwdByAdmin">
```
Comment (harishbnaritha) [inline apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/WEB-INF/controller.xml]: lets call it as initResetEmployeePwd
employee in admin means a tenant user

File: apache-ofbiz-16.11.05/hot-deploy/admin/src/main/java/com/autopatt/admin/events/PasswordMgmtEvents.java [hunk 2]
```diff
@@ -88,4 +90,31 @@ public static String resetPassword(HttpServletRequest request, HttpServletRespon
         }
         return SUCCESS;
     }
+
+    public static String sendPasswordResetLinkByAdmin(HttpServletRequest request, HttpServletResponse response) {
```
Comment (harishbnaritha) [inline apache-ofbiz-16.11.05/hot-deploy/admin/src/main/java/com/autopatt/admin/events/PasswordMgmtEvents.java]: lets call this resetEmployeePassword

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_employees.ftl [hunk 3]
```diff
@@ -165,9 +166,11 @@
                 <br/>
                 <small>An email will be sent to user with a link to set their password.</small>
             </div>
+            <input type="hidden" name="resetPasswordUserLoginId" id="resetPasswordUserLoginId" value=""/>
+            <input type="hidden" name="resetPasswordOrgPartyId" id="resetPasswordOrgPartyId" value=""/>
             <div class="modal-footer">
                 <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
-                <button type="button" class="btn btn-primary">Reset Password</button>
+                <button type="button" class="btn btn-primary" onclick="initResetPwdByAdmin()">Reset Password</button>
```
Comment (harishbnaritha) [inline apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_employees.ftl]: similar naming convention, lets consider tenant users as employees

File: apache-ofbiz-16.11.05/hot-deploy/admin/src/main/java/com/autopatt/admin/events/PasswordMgmtEvents.java (hunk 2 above)
Comment (RavindraDesai042) [reply apache-ofbiz-16.11.05/hot-deploy/admin/src/main/java/com/autopatt/admin/events/PasswordMgmtEvents.java]: but we are not resetting here, we are justing sending the link with token

Comment (harishbnaritha) [reply apache-ofbiz-16.11.05/hot-deploy/admin/src/main/java/com/autopatt/admin/events/PasswordMgmtEvents.java]: ok we will call yet sendEmployeePasswordResetLink

**Pull request merged**